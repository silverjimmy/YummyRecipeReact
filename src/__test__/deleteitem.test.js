import React from 'react';
import ReactDOM from 'react-dom';
import { render, configure, shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import expect from 'expect';
import Adapter from 'enzyme-adapter-react-15';
import DeleteRecipe from '../components/deleteitem';
import LocalStorageMock from '../setupTests';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

const match = {
    params: '1',
};

const setUp = () => {
    const props = {
        match,
    };
    return shallow(<DeleteRecipe {...props} />);
};

describe('Createcategory', () => {
    it('matches  snapshot', () => {
        const deleteRecipe = setUp;
        expect(shallowToJson(deleteRecipe)).toMatchSnapshot();
    });

    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(
            <MuiThemeProvider>
                <DeleteRecipe />
            </MuiThemeProvider>,
            div,
        );
    });
});
