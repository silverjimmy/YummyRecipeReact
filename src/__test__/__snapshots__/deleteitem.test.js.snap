// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Createcategory matches  snapshot 1`] = `null`;
